src/libraries/Microsoft.Win32.Registry/Directory.Build.props

```diff
@@ -2,6 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
Do we gain any benefit from annotating what we expect to be in the framework vs not?  What if we just stuffed everything in here that wasn't part of the base shared framework?

Reply:
> Do we gain any benefit from annotating what we expect to be in the framework vs not? 

did u mean putting all the oob assemblies in this package ? We will still have to maintain a list of libraries which are part of aspnetcore shared or windowsDesktop shared framework so that we dont rev up the assembly version

Reply:
IMO I prefer to be explicit here and only add what is required by aspnetcore. This is not just about file size of the package (which has an impact on CI) but also about correctness.

Reply:
> We will still have to maintain a list of libraries which are part of aspnetcore shared or windowsDesktop shared framework so that we dont rev up the assembly version

I see.  In that case we should be explicit everywhere.  I agree we should remove the wildcard for extensions too.  That was helpful when doing the original port but no longer.  We should also remove any of the other special casing for extensions in the props/targets.